docs/migration-guides/bkpr-1.2-migration-guide.md

```diff
@@ -0,0 +1,52 @@
+# Migration Guide: Moving to BKPR v1.2
+
+This document is intended towards users who want upgrade their Kubernetes clusters to BKPR v1.2 from BKPR v1.1 and below. The document will assist in migrating a Kubernetes cluster to BKPR v1.2. Be sure to read this guide in its entirety before performing the upgrade.
```

Comment:
```suggestion
This document is for users who want to upgrade their Kubernetes clusters to BKPR v1.2 from BKPR v1.1 and below. The document will assist in migrating a Kubernetes cluster to BKPR v1.2. Be sure to read this guide in its entirety before performing the upgrade.
```

Reply:
Aside: I think we should remove "or below" from this doc.  As discussed earlier, it will be much easier if we _only_ need to handle upgrades from v1.1->v1.2.  (Easy enough to manage both 1.0 and 1.1 in this case, but this will be increasingly difficult going forward.  We can only realistically support/document/test strict N -> N+1 upgrades at this time, so we may as well start saying that now.)